src/modules/packages.py

```diff
@@ -2,14 +2,14 @@
 
 commands = [
     "pacman -Qq --color never",  # Arch Linux
-    "xbps-query -l",  # Void Linux
-    "kiss l",  # KISS Linux
     "dpkg-query -f '.\n' -W",  # Debian, Ubuntu, Mint
-    "dnf list installed",  # Fedora
-    "zypper search -i",  # openSUSE
+    "dnf list installed | tail -n +2",  # Fedora, RHEL
     "rpm -qa",  # RHEL, Fedora Core, CentOS
-    "yum list installed",  # RHEL, Fedora Core, CentOS
+    "yum list installed | tail -n +2",  # RHEL, Fedora Core, CentOS
+    "xbps-query -l",  # Void Linux
     "nix-store -qR /run/current-system/sw",  # NixOS
```

Comment:
Can we consider moving nix to the bottom or to the top in the list (or making a version that lets us use multiple package managers)

Nix is installable on non-nixos systems so it'd be nice to have parity between what happens when nix is installed

Reply:
I'd say it makes sense to be on the bottom of the list - people will expect package count to be the system package manager by default. 

However in the case someone uses Nix way more than the system package manager we might want to add a better way of determining packages, such as a flag to force using Nix, then it could be aliased to always force Nix package counts. But that is a separate issue and probably needs a separate PR.